.github/workflows/sonarcloud.yml

```
@@ -10,39 +10,73 @@
+          echo "DB_NAME=sonar" >> $GITHUB_ENV
+          echo "DB_USER=sonar" >> $GITHUB_ENV
+          echo "DB_PASSWORD=sonar" >> $GITHUB_ENV
+
       - name: Set up Python
         uses: actions/setup-python@v4
         with:
           python-version: 3.12.4
 
-      # Set up Node.js and Vitest (Frontend)
       - name: Set up Node.js
         uses: actions/setup-node@v3
```

actionlint (1.7.4)

the runner of "actions/setup-node@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)